client/src/ui/modules/ProfilePage.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useMemo, useCallback } from "react";
+import {
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/ui/elements/dialog";
+import { CardContent } from "@/ui/elements/card";
+import { Button } from "@/ui/elements/button";
+import { Achievements } from "./Achievements";
+import { Statistics } from "./Statistics";
+import Connect from "../components/Connect";
+import { useGames } from "@/hooks/useGames";
+import { usePlayer } from "@/hooks/usePlayer";
+import useAccountCustom, { ACCOUNT_CONNECTOR } from "@/hooks/useAccountCustom";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/ui/elements/tabs";
+import { motion } from "framer-motion";
+import { Game } from "@/dojo/game/models/game";
+
+// Constantes pour les seuils de score et de combo
+const SCORE_THRESHOLDS = [50, 100, 150, 200, 250, 300, 350, 400, 450, 500];
+const COMBO_THRESHOLDS = [5, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100];
```

Comment:
Those variables are already implemented here: client/src/dojo/game/types/level.ts
use them instead if you can

Reply:
Alright.
For the combo, I'll set a step of 5.